governance.md

```diff
@@ -197,56 +218,219 @@ interests include, but are not limited to:
+position by majority vote.
+
+Any currently active Contributor is eligible to be considered for a DPL and an
+individual may hold more than one DPL simultaneously.
+
+
+### Release Manager(s)
+
+The Release Manager (RM) is appointed for a minor version (A.B.x) release series
+of Matplotlib rather than for a fixed term.  They are responsible for the full
+release life cycle of all minor releases in the series including:
+
+- ensuring the whats new, API changes, and release notes are up to date
+- the timing of the releases
+- what changes should or should not be backported from the master
+  branch
+- rebuilding and publishing the website
+- publishing sdist and wheels to pypi
+- notifying down-stream packagers of the release
+- announcing the release (in coordination with the Community Manager)
+
+An individual may be the RM as more than one release series at the
+same time.
+
+
+### API consistency Leader
+
+Matplotlib is constantly making small changes to our API: enhancements
+that add new features, bug fixes that unavoidably change behavior, and
+deprecation of inconsistent or undesired functionality.  The API
+Consistency Leader (ACL) is responsible for making sure that these
+incremental changes to the library are done in a coherent and
+consistent manner.
+
+This include checking that:
+
+- new functionality is not duplicating existing functionality
+- deprecations are justified and properly documented
+- new functionality does not "paint us into a corner" for future work
+- new functionality is implemented with an API that is consistent with
+  the existing functions
+
+
+### Reference Documentation Leader
+
+The Matplotlib API reference documentation is split between the docstrings and
+the rst source.  This documentation needs to be complete and accurate as our
+users rely on it as the final authority of what a given method will do (short of
+reading the source).
+
+The Reference Documentation Leader (RDL) is responsible for ensuring that
+the docstrings are:
+
+- correctly formatted and render as intended
+- technically correct
+- complete
 
+In addition to the docstrings the RDL is responsible for the sphinx build
+machinery and our sphinx extensions.
+
+### Narrative Documentation Leader
+
+In addition to the refernce documentation, Matplotlib has narrative documentation.
+This documentation can take the form of short "cookbook" examples that show an
+end-to-end
+
+
+This includes our examples and tutorials that live inside the
+main source repository and longer tutorials that live in other repositories
+in the Matplotlib organization.
+
+The Narrative Documentation Leader is responsible for shepherding all of this
+content including the scope, level, tone, and voice.
+
+### Secretary
+
+- Responsible for ensure that there is an agenda for the weekly meeting at it is
+  followed.
+- Responsible for maintaining the weekly notes.
+- Responsible for ensuring that decisions taken by vote in the SC are properly
```

Comment:
This is a confusing responsibility, unless the Secretary will be part of the SC.

Reply:
Good point, moved this to PL.